### ISSUE TITLE
npm list returns dep missing

### ISSUE THREAD

**Issue by tuandm** [Rias500/laravel-mix-eslint#4]
I got the error with laravel-mix 4.0.7 when installing this plugin:
```
npm ERR! peer dep missing: laravel-mix@^2.1.7, required by laravel-mix-eslint@0.1.2
```
I'm not sure the correct way to fix this, but changing the laravel-mix version in package.json worked well. Hope you can give the best solution.

**Comment by Rias500** [comment]
Ah yes, Mix 4.0 and 3.0 are major releases, which this doesn't account for. i'll update the package.json

**Comment by Rias500** [comment]
Should be fixed when using laravel-mix-eslint 0.1.3
